### ISSUE TITLE
Create consistent headings for Local Instructions

### ISSUE THREAD

**Issue by alphadelta332** [vatpac-technology/sops#325]
Discuss having the same sets of headings for each type of LI (Tower, Approach, Enroute), consistent across all

**Comment by alphadelta332** [comment]
Aerodromes
- Positions
- Airspace
- Maneuvering Area
    - Responsibility
    - Standard Taxi Routes
    - Taxiway Restrictions
- Separation
    - Responsibility
    - Lateral Separation Points (applicable to Procedural Towers)
- Local Procedures (eg Scenic Coastals, Special Military Procedures)
- VFR Operations
    - Departure/Arrival Gates
- Helicopter Operations
- Runway Modes
    - Runway Selection
    - Special Runway Operations (eg PROPS, SODPROPS)
    - Circuits
- SID Selection
- ATIS
- Coordination
- Charts (mainly applicable to Military ADs)

**Comment by alphadelta332** [comment]
**TCUs**
- Positions
- Airspace
    - Reclassifications
    - Airspace Division
    - Special Positions
- Separation
    - Responsibility
- Local Procedures (eg Scenic Coastals, Special Military Procedures)
- Departure/Arrival Procedures
- Tower Offline Procedures
- Runway Modes
    - Parallel Runway Operations
- Helicopter Operations
- Flow
- Coordination

**Comment by alphadelta332** [comment]
**Enroute**
- Positions
- Airspace
    - Reclassifications
    - Special Positions
- Extending
- CPDLC
- Sector Responsibilities
    - Sequencing
- Runway Modes
- Surveillance Coverage
- STAR Clearances
- STAR Clearance Expectation
- Coordination

**Comment by mattkelly4** [comment]
> **Aerodromes**
> 
> * Positions
> * Airspace
> * Maneuvering Area
>   
>   * Responsibility
>   * Standard Taxi Routes
>   * Taxiway Restrictions
> * Separation
>   
>   * Responsibility
>   * Lateral Separation Points (applicable to Procedural Towers)
> * Local Procedures (eg Scenic Coastals, Special Military Procedures)
> * VFR Operations
>   
>   * Departure/Arrival Gates
> * Helicopter Operations
> * Runway Modes
>   
>   * Runway Selection
>   * Special Runway Operations (eg PROPS, SODPROPS)
>   * Circuits
> * SID Selection
> * ATIS
> * Coordination
> * Charts (mainly applicable to Military ADs)

This looks great. Thoughts on nesting **Separation** under **Airspace**? And maybe **Charts** under **Local Procedures** (since it's relevant to the lack of certain charts in the AIP for that specific military location)?

**Comment by alphadelta332** [comment]
> This looks great. Thoughts on nesting **Separation** under **Airspace**? And maybe **Charts** under **Local Procedures** (since it's relevant to the lack of certain charts in the AIP for that specific military location)?

Hmmmm. I think Separation (responsibilities, etc) are distinct enough from airspace (structure, ownership, runway modes, etc) to be their own separate categories.

Yeah maybe Charts could go as like a subsubcategory, ie Local Procedures > Military Operations > Charts, or something

**Comment by mattkelly4** [comment]
> > This looks great. Thoughts on nesting **Separation** under **Airspace**? And maybe **Charts** under **Local Procedures** (since it's relevant to the lack of certain charts in the AIP for that specific military location)?
> 
> Hmmmm. I think Separation (responsibilities, etc) are distinct enough from airspace (structure, ownership, runway modes, etc) to be their own separate categories.
> 
> Yeah maybe Charts could go as like a subsubcategory, ie Local Procedures > Military Operations > Charts, or something

Yeah fair, I was thinking that separation requirements relate to airspace ownership, but I'm happy to keep separate.

**Comment by mattkelly4** [comment]
@alphadelta332 thoughts on a section on each aerodrome page for where each operator parks? Could just be a diagram which might be helpful when someone says they want to taxi to 'the Qantas bays'
